src/index.js

```diff
@@ -15,5 +15,10 @@ module.exports = function({
   verbose && setVerboseMode();
   switchOff && setSwitchOff();
 
-  return { trace: trace({ token, debug, edgeHost, switchOff, eventFilter }) };
+  return {
+    trace: trace({ token, debug, edgeHost, switchOff, eventFilter }),
+    reportError: msg => {
```

Comment:
In case they use our plugin then they will need to import our tracer and initialize it to use this function. Not good. Please do it as you did in Python, a separate function